admin-dev/themes/new-theme/js/pages/product/components/generator/AttributesSelector.vue

```diff
@@ -120,31 +120,21 @@
 </template>
 
 <script lang="ts">
-  import Vue, {PropType, VueConstructor} from 'vue';
+  import {defineComponent, PropType} from 'vue';
   import isSelected from '@pages/product/mixins/is-attribute-selected';
   import ProductMap from '@pages/product/product-map';
   import PerfectScrollbar from 'perfect-scrollbar';
   // @ts-ignore
   import Bloodhound from 'typeahead.js';
   import AutoCompleteSearch, {AutoCompleteSearchConfig} from '@components/auto-complete-search';
   import Tokenizers from '@components/bloodhound/tokenizers';
-  import {Attribute, AttributeGroup} from '@pages/product/types';
+  import {Attribute, AttributeGroup, AttributesSelectorStates} from '.';
 
   const {$} = window;
 
   const CombinationsMap = ProductMap.combinations;
 
-  export interface AttributesSelectorStates {
```

Comment:
Not sure this type is useful anymore... It was meant to type correctly this Vue app state, but although you moved it in the index file (which I don't really agree as well) It is not used anymore so it could be removed in the end